L/LAST/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "LAST"
+version = v"1499"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://gitlab.com/mcfrith/last.git",
+        "2cc68d3ba8ae5ca46ceeb69302aef18b9db04f46",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/last
+install_license COPYING.txt
+export CXXFLAGS=" -O3 -Wall -g -std=c++11 -pthread"
+if [[ "${target}" != aarch64-* ]]; then
+    CXXFLAGS="${CXXFLAGS} -msse4"
```

Comment:
Is there a reason you are manually setting the `sse4` architecture flag?

Reply:
It's what was written in their Makefile, I just copy/pasted. I definitely need this program to be vectorized + they rely on SSE4 intrinsics directly. Unless GCC can automatically enable `-msse4`?

Reply:
I agree in general we don't want to do stuff like this for generic binaries, but in this specific case it may not be _too_ bad: sse4 is fairly old and if it makes a big difference for this package it could be beneficial to enable the extension here.